chainercv/datasets/ade20k/ade20k_semantic_segmentation_dataset.py

```diff
@@ -0,0 +1,103 @@
+import glob
+from multiprocessing import Pool
+import os
+
+import numpy as np
+
+from chainer import dataset
+from chainer.dataset import download
+from chainercv import utils
+from chainercv.utils import read_image
+
+root = 'pfnet/chainercv/ade20k'
+trainval_url = 'http://data.csail.mit.edu/places/ADEchallenge/'
+trainval_url += 'ADEChallengeData2016.zip'
+test_url = 'http://data.csail.mit.edu/places/ADEchallenge/release_test.zip'
+
+
+def get_ade20k():
+    p = Pool(2)
+    data_root = download.get_dataset_directory(root)
+    urls = [trainval_url, test_url]
+    ret = [p.apply_async(utils.cached_download, args=(url,)) for url in urls]
```

Comment:
`cached_download` shows an progress report. Is it safe to call multiple `cached_download` at the same time?

Reply:
Right. So, actually, it tries to show two progress reports at the same line, then the display of the progress report is broken. Should I do those download sequentially?

Reply:
I prefer sequential downloading. Is the downloading too slow?